pkgs/applications/office/elementary-planner/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, appstream-glib, desktop-file-utils
+, python3, vala, wrapGAppsHook
+, evolution-data-server
+, libical
+, libgee
+, json-glib
+, sqlite
+, libsoup
+, gtk3
+, pantheon /* granite, schemas */
+, webkitgtk
+, appstream
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elementary-planner";
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "alainm23";
+    repo = "planner";
+    rev = version;
+    sha256 = "0swj94pqf00wwzsgjap8z19k33gg1wj2b78ba1aj9h791j8lmaim";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    meson
+    ninja
+    pkgconfig
+    python3
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    evolution-data-server
+    libical
+    libgee
+    json-glib
+    sqlite
+    libsoup
+    gtk3
+    pantheon.granite
+    webkitgtk
+    appstream
```

Comment:
where do we need appstream?

Reply:
Possibly for appstream It's file for merging the localisations into appstream file?

Reply:
Don't remember! Possibly sources:

* I've been tracking this since before it was usable, maybe it needed/asked for it previously and I missed the drop
* The in-tree `debian/control` mentions `appstream` in `Build-Depends:` so maybe based initial deps on that list
* I maybe copied from another expression that does need appstream :3.

In short, thank you for checking these and pushing back on unnecessary deps-- this has been working so I forgot I put it together rather sloppily.  Sorry about that :3.

FWIW I believe `libical` isn't asked for when a sufficiently new `libecal` (`evolution-data-server`) but I provided anyway so this works with, for example, my tree which has yet to om nom the gnome 3.34 updates O:).

But I'm happy to remove it, which is why I bring it up ;).

Reply:
`git log -p -Gappstream` suggests it was used for things like `appstreamcli validate` on the appdata.xml or so?

Removing, will see how that pans out :).